flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -204,12 +192,16 @@ object AkkaUtils {
       |
       |  loggers = ["akka.event.slf4j.Slf4jLogger"]
       |  logger-startup-timeout = 30s
-      |  loglevel = "WARNING"
-      |  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+      |  loglevel = "DEBUG"
```

Comment:
Well actually, it is not optimal, since now all debug messages are generated but not printed. I'll fix it.